Add TailwindCSS and improve styling

JiaHongL opened the pull request:
before

<img width="372" alt="image" src="https://github.com/tianci-sh/BreezeAds/assets/6601449/9f341d0e-2f87-46bb-a326-687c03156cfa">

after

<img width="413" alt="image" src="https://github.com/tianci-sh/BreezeAds/assets/6601449/b7d5f803-540f-4d2a-8a99-1b4f19ed95aa">

vocolboy commented on src/chrome/main.html:
```diff
@@ -1,92 +1,41 @@
+	<meta charset="UTF-8" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+	<title>BreezeAds</title>
+	<link href="https://unpkg.com/tailwindcss@^2.0/dist/tailwind.min.css" rel="stylesheet" />
```
It can't use remote code (additional explanation needed). 
It may need to move to a local 'assets' folder.

JiaHongL replied:
The local file has been added

JiaHongL commented:
Added local tailwindcssv2.2.19.css

PS: Since Tailwind v3 uses JIT mode, package.json and tailwind.config.js need to be configured. Therefore, we switched to using Tailwind v2.
